Misc/NEWS

```diff
@@ -32,6 +32,9 @@ Core and Builtins
   anymore.  It affects ``co_firstlineno`` and ``co_lnotab`` of code object
   for module and class.
 
+- bpo-29683: Fixes to memory allocation in _PyCode_SetExtra. Patch by
```

Comment:
New entries usually added are added at the start of corresponding section, so that entries are list in reversed chronological order.

It is recommended to use two spaces after sentence-ending period.